examples/p4est_3d_dgsem/elixir_euler_baroclinic_instability.jl

```diff
@@ -190,7 +238,35 @@ semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver,
 # ODE solvers, callbacks etc.
 
 tspan = (0.0, 10 * 24 * 60 * 60.0) # time in seconds for 10 days
-ode = semidiscretize(semi, tspan)
+
+# Save RHS of the steady state and subtract it in every RHS evaluation.
+# This trick preserves the steady state exactly (to machine rounding errors, of course).
+# Otherwise, this elixir produces entirely unusable results for a resolution of 8x8x4 cells
+# per cube face with a polydeg of 3.
+# With this trick, even the polydeg 3 simulation produces usable (although badly resolved) results,
+# and most of the grid imprinting in higher polydeg simulation is eliminated.
+#
+# See https://github.com/trixi-framework/Trixi.jl/issues/980 for more information.
+u_steady_state = compute_coefficients(steady_state_baroclinic_instability, 0.0, semi)
+# Use a `let` block for performance (otherwise du_steady_state will be a global variable)
+let du_steady_state = similar(u_steady_state)
+  # Save RHS of the steady state
+  Trixi.rhs!(du_steady_state, u_steady_state, semi, 0.0)
```

Comment:
```suggestion
u_steady_state = compute_coefficients(steady_state_baroclinic_instability, tspan[1], semi)
# Use a `let` block for performance (otherwise du_steady_state will be a global variable)
let du_steady_state = similar(u_steady_state)
  # Save RHS of the steady state
  Trixi.rhs!(du_steady_state, u_steady_state, semi, tspan[1])
```